src/uncss.js

```diff
@@ -213,11 +213,12 @@ function init(files, options, callback) {
         media: [],
         timeout: 0,
         report: false,
-        ignoreSheets: []
+        ignoreSheets: [],
+        httpSetting: {},

```

Comment:
Also, FYI, the project's linting rules failed the CI tests because there's a trailing comma on this line